### PR TITLE
Libxslt

### PR DESCRIPTION
**Pull request by DemiMarie**

### CODE REVIEW

File: contrib/libxslt/libxslt_sapi.h [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        .AllowOpen()
+        .AllowRead()
+        .AllowStat()
+        .AllowWrite()
+        .AllowExit()
+        .AllowSyscalls({
+            __NR_futex,
+            __NR_close,
+            __NR_lseek,
+            __NR_getpid,
```
Comment (okunz) [inline contrib/libxslt/libxslt_sapi.h:44]: NIT: Replace __NR_getpid & __NR_gettid with AllowGetPID() and __NR_gettimeofday with AllowTime()

File: contrib/libxslt/sapi_libxslt_test.cc [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+#include <libxslt/xslt.h>
+#include <libxslt/xsltInternals.h>
+#include <libxslt/xsltutils.h>
+
+#include "libxslt_sapi.h"
+
+namespace {
+
+using ::sapi::IsOk;
+using ::testing::Eq;
```
Comment (okunz) [inline contrib/libxslt/sapi_libxslt_test.cc:44]: Remove unused matchers: Eq, Gt, Not, and StrEq

**Review by okunz — COMMENTED** [review]
Can we separate the libxslt changes and the changes to the other libraries that seem unrelated?

File: contrib/libxslt/sapi_libxslt_test.cc [hunk 3]
```diff
@@ -0,0 +1,121 @@
+/*
+ * libxslt_tutorial.c: demo program for the XSL Transformation 1.0 engine
+ *
+ * based on xsltproc.c, by Daniel.Veillard@imag.fr
+ * by John Fleck
+ *
+ * See Copyright for the status of this software.
+ *
+ */
```
Comment (okunz) [inline contrib/libxslt/sapi_libxslt_test.cc:9]: Please replace with our standard header (same in sample_clsproc.cc)

Comment (DemiMarie) [reply contrib/libxslt/sapi_libxslt_test.cc:9]: This code is derived from the XSLTproc source code.  That code is under an MIT license so there should be no problem using it, but the copyright notice and license text will need to be included somewhere.

Comment (okunz) [reply contrib/libxslt/sapi_libxslt_test.cc:9]: In an internal discussion we thought it's better to use our header in this instance.

Comment (cblichmann) [reply contrib/libxslt/sapi_libxslt_test.cc:9]: I think I should've been clearer, I actually meant having both, especially the original author. The changes to the file will be under Google copyright.

**Pull request closed without merging**